.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -84,6 +84,14 @@ dynamodb_CreateTable:
             - description:
               snippet_tags:
                 - dynamodb.rust.create-table
+    Swift:
+      versions:
+        - sdk_version: 1
+          github: swift/example_code/ddb/basics

```

Comment:
This should be swift/example_code/ddb -- point to the service folder, not the scenario.